### ISSUE TITLE
v11: pfperl-api doesn't start

### ISSUE THREAD

**Issue by nqb** [inverse-inc/packetfence#6498]
**Describe the bug**
Integration tests failed last night at configurator step (on EL8 and Debian 11), almost all API calls returned 502 error code.

This was not the case yesterday. I think it's caused by following commit: 8fc9df6ed4f58904a4fd29bc985d5394114b39c7

Logs:
- https://gitlab.com/inverse-inc/packetfence/-/jobs/1493834862#L497
- https://gitlab.com/inverse-inc/packetfence/-/jobs/1493834870#L472

**Comment by nqb** [comment]
Logs of `pfperl-api`:

```
Aug 11 03:42:45 pfdeb11dev packetfence[26871]: pfperl-api(26871) FATAL: unable to connect to database: Access denied for user 'pf'@'localhost' at /usr/local/pf/lib/pf/dal.pm line 70.
                                               Compilation failed in require at /usr/local/pf/lib/pfappserver/Form/Config/Profile.pm line 16.
                                               BEGIN failed--compilation aborted at /usr/local/pf/lib/pfappserver/Form/Config/Profile.pm line 16.
                                               Compilation failed in require at /usr/local/pf/lib/pf/UnifiedApi/Controller/Config/ConnectionProfiles.pm line 21.
                                               BEGIN failed--compilation aborted at /usr/local/pf/lib/pf/UnifiedApi/Controller/Config/ConnectionProfiles.pm line 21.
                                               Compilation failed in require at (eval 3387) line 1.
                                                (Mojo::Loader::load_classes)

```

**Comment by jrouzierinverse** [comment]
Fixed by 3502c291b189a502b89f87767f71db12d81d3895

**Comment by nqb** [comment]
and also https://github.com/inverse-inc/packetfence/commit/3502c291b189a502b89f87767f71db12d81d3895
